### ISSUE TITLE
Copy text in the Log

### ISSUE THREAD

**Issue by yanavasileva** [camunda/camunda-modeler#1380]
__Describe the Bug__

I'm not able to copy a snippet (an error message which occurred) from the Modeler's log.

__Steps to Reproduce__

1. Open Log View
2. Mark part of the text
3. Ctrl + C

__Expected Behavior__

The marked text is saved to the Clipboard.

__Environment__

Please complete the following information:

 - OS: Windows 10
 - Camunda Modeler Version: 3.1.0

**Comment by yanavasileva** [comment]
If I need to provide additional information (or it a feature request and not a bug), please let me know.

**Comment by pinussilvestrus** [comment]
Thanks for reporting!

It is already known, that copy-pasting does not work directly inside the Log panel since we're not handling the keyboard events inside it. We should definitely tackle this.

One thing you can do is to use the Copy button to copy _all_ the contents

![May-22-2019 14-56-57](https://user-images.githubusercontent.com/9433996/58176113-e0867d80-7ca1-11e9-9ccf-f85e5d34631a.gif)

**Comment by yanavasileva** [comment]
Hi @pinussilvestrus,

Thank you for your reply.
I am aware of the option to copy everything, however, it will be helpful to copy just part of it.

Best,
Yana

**Comment by nikku** [comment]
You can properly copy as of Camunda Modeler v3.6.

**Comment by pinussilvestrus** [comment]
What's described in the issue is not working for me on macOS version 10.15.2, Modeler Version v3.6

Assuming "Test" is in my clipboard at the beginning, marking some text in the Log and Pressing CMD+C.

![Kapture 2020-02-11 at 9 30 01](https://user-images.githubusercontent.com/9433996/74220937-2f04fd80-4cb1-11ea-9f02-e12d1572dd86.gif)

**Comment by nikku** [comment]
So seems to be an issue exclusively on MacOS nowadays.

**Comment by pinussilvestrus** [comment]
On macOS, we need to have the `role` properties given for all native keyboard shortcuts.
```sh
{
  [
      { role: 'undo' },
      { role: 'redo' }
   ],
   [
      { role: 'cut' },
      { role: 'copy' },
      { role: 'paste' },
      { role: 'selectall' }
    ]
  }
}
```
 We're overriding them when loading the editors. 

So we basically need something like we implemented for modals: https://github.com/camunda/camunda-modeler/commit/e432ff787bee57f612c0b090c9178db6713863d7

**Comment by pinussilvestrus** [comment]
Closes via https://github.com/camunda/camunda-modeler/commit/6e304d9e23f865e36f3641d4a91bdf84aff489d1
